annotation/core/models.py

```diff
@@ -71,15 +71,21 @@ def boundary(self):
     def __str__(self):
         return self.body
 
-
+class FeedbackOption(models.Model):
+    """The types of reasons people give for thinking sentence is machine generated text"""
+    shortname = models.CharField(max_length=100)
+    description = models.CharField(max_length=250)
```

Comment:
Let's add one more field here:
```
is_default = models.BooleanField(default=True)
```
This will be True for our main options and False for all the user-specified ones.

Reply:
added